bst_plugin/process_nst_dOD_SSR.m

```diff
@@ -0,0 +1,299 @@
+function varargout = process_nst_dOD_SSR( varargin )
```

Comment:
If I understand well, this function simply adds another scenario to dOD computation.
So you can directly modify `process_nst_dOD` while keeping the default behavior to the original scenario.
Let's not duplicate too much code ;)